core/CMakeLists.txt

```diff
@@ -74,6 +74,14 @@ else()
   target_link_libraries(${LINK_LIBRARIES})
 endif ()
 
+# needed to SQLITECPP to work for ndk15c+ and android api level < 21
+# refer:
+# https://github.com/android-ndk/ndk/issues/477 and
+# https://android.googlesource.com/platform/bionic/+/master/docs/32-bit-abi.md
+if (ANDROID)
+    add_definitions(-D_FILE_OFFSET_BITS=32)
```

Comment:
Can we add this flag to just the SQLiteCpp project and not the whole core library? We should be able to add it here, just like this other definition: https://github.com/tangrams/tangram-es/blob/master/core/deps/CMakeLists.txt#L79

Reply:
Sure, will update